From 8cd1103c112b415f8853ad719358413cf9cbf93a Mon Sep 17 00:00:00 2001
From: Aravinda VK <avishwan@redhat.com>
Date: Thu, 16 Jul 2015 18:23:36 +0530
Subject: [PATCH 112/131] tools/glusterfind: Do not show session corrupted if no status file

When a glusterfind session is created it creates session directories
in all the nodes which are part of the Volume. But session status
file only present in initiated node.

Show Session corrupted only if status file exists and invalid content.

Change-Id: I8443c0335b872645b54c0aa77d9893dbe2589c92
Signed-off-by: Aravinda VK <avishwan@redhat.com>
BUG: 1224880
Reviewed-on: http://review.gluster.org/11699
Reviewed-on: http://review.gluster.org/12028
Reviewed-by: Milind Changire <mchangir@redhat.com>
Reviewed-by: Saravanakumar Arumugam <sarumuga@redhat.com>
Reviewed-by: Kotresh Hiremath Ravishankar <khiremat@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/61779
---
 tools/glusterfind/src/main.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/glusterfind/src/main.py b/tools/glusterfind/src/main.py
index 9a6a2ac..b4d2c7e 100644
--- a/tools/glusterfind/src/main.py
+++ b/tools/glusterfind/src/main.py
@@ -519,7 +519,7 @@ def mode_list(session_dir, args):
                     last_processed = f.read().strip()
             except (OSError, IOError) as e:
                 if e.errno == ENOENT:
-                    pass
+                    continue
                 else:
                     raise
             output.append((session, volname, last_processed))
-- 
1.7.1

